### PR TITLE
Feature build

### PR DESCRIPTION
**Pull request by bcleveland1022**
Fixes #
Low side site build.
## Proposed Changes

  -
  -
  -

### CODE REVIEW

File: open-sphere-plugins/imagery/src/main/java/io/opensphere/imagery/ImageryReimportDataCreator.java [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+import io.opensphere.mantle.data.DataGroupInfo;
+import io.opensphere.mantle.datasources.IDataSource;
+import io.opensphere.mantle.datasources.IDataSourceCreator;
+
+/**
+ * Creates an imagery data source on reimport.
+ */
+public class ImageryReimportDataCreator implements IDataSourceCreator
+{
+
```
Comment (rkausch) [inline open-sphere-plugins/imagery/src/main/java/io/opensphere/imagery/ImageryReimportDataCreator.java]: blank line

File: open-sphere-plugins/imagery/src/main/java/io/opensphere/imagery/ImageryImportDataCreator.java [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+import io.opensphere.mantle.data.DataGroupInfo;
+import io.opensphere.mantle.datasources.IDataSource;
+import io.opensphere.mantle.datasources.IDataSourceCreator;
+
+/**
+ * Creates an imagery data source on import.
+ */
+public class ImageryImportDataCreator implements IDataSourceCreator
+{
+
```
Comment (rkausch) [inline open-sphere-plugins/imagery/src/main/java/io/opensphere/imagery/ImageryImportDataCreator.java]: blank line
```suggestion```

File: open-sphere-plugins/imagery/src/main/java/io/opensphere/imagery/ImageryReimportDataCreator.java (hunk 1 above)
Comment (bcleveland1022) [reply open-sphere-plugins/imagery/src/main/java/io/opensphere/imagery/ImageryReimportDataCreator.java]: Removed blank line.

File: open-sphere-plugins/imagery/src/main/java/io/opensphere/imagery/ImageryImportDataCreator.java (hunk 2 above)
Comment (bcleveland1022) [reply open-sphere-plugins/imagery/src/main/java/io/opensphere/imagery/ImageryImportDataCreator.java]: Removed blank line.

**Pull request merged**